soap/src/java/com/zimbra/soap/mail/type/FreeBusyUserSpec.java

```diff
@@ -51,8 +56,11 @@
     public FreeBusyUserSpec() {
     }
 
+    @GraphQLInputField(name=GqlConstants.FOLDER_ID, description="Calendar folder ID; if omitted, get f/b on all calendar folders")
     public void setFolderId(Integer folderId) { this.folderId = folderId; }
+    @GraphQLInputField(name=GqlConstants.ID, description="Zimbra ID Either name or id must be specified")
```

Comment:
instead of using this class, better to create new one in zm-gql/models/input and in that we can have selector (byName or byId) and the value.